M2/Macaulay2/d/hashtables.dd

```diff
@@ -340,7 +340,12 @@ export lookup1force(object:HashTable,key:Expr,keyhash:hash_t):Expr := (
 	  bucket = bucket.next;
 	  );
      if lockit then unlock(object.mutex);
-     buildErrorPacket("key not found in hash table"));
+     msg := "key not found in hash table";
+     when key
+     is x:stringCell do msg = msg + ": \"" + x.v + "\""
+     is x:SymbolClosure do msg = msg + ": " + x.symbol.word.name
+     else nothing;
+     buildErrorPacket(msg));
```

Comment:
I'm not sure if you still want to go ahead with this PR (I'm curious to see Paul's version also), but for the record, I think this chunk should be in a separate function, say `MissingKeyError(object,key)`. Then, for instance, it would be easy for Paul's version to call a top-level function for printing the missing key.